test/integration/nixos.nix

```diff
@@ -57,22 +60,51 @@ pkgs.testers.runNixOSTest {
       };
     };
   };
+in
+  pkgs.testers.runNixOSTest {
+    name = "pareto";
 
-  interactive.nodes.machine = {...}: {
-    services.openssh.enable = true;
-    services.openssh.settings = {
-      PermitRootLogin = "yes";
-      PermitEmptyPasswords = "yes";
+    nodes.machine_gnome = {
+      pkgs,
+      lib,
+      ...
+    }: {
+      imports = [makeParetoBaseConfig];
+
+      # Install GNOME Desktop Environments
+      services.xserver.desktopManager.gnome.enable = true;
+      services.xserver.displayManager.gdm.enable = true;
+    };
+
+    nodes.machine_kde = {
+      pkgs,
+      lib,
+      ...
+    }: {
+      imports = [makeParetoBaseConfig];
+
+      # Install KDE Plasma 5 Desktop Environment
+      services.xserver.enable = true;
+      services.xserver.desktopManager.plasma5.enable = true;
+      services.xserver.displayManager.sddm.enable = true;
+      services.colord.enable = false;
+    };
+
+    interactive.nodes.machine = {...}: {
```

Comment:
This sould probably not be committed?

Reply:
We already have this - https://github.com/ParetoSecurity/agent/blob/main/test/integration/nixos.nix#L61 .

If we want to keep it I need to update the names, instead of nodes.machine it should be nodes.machine_gnome and nodes.machine_kde.

Reply:
Actually, you can do whatever fits best at the moment. When we have more than two or three tests in the repo, we'll pull out the interactive config into a common file, so that it's a one-liner to use in tests.